### ISSUE TITLE
Transform not resetting when closing drawer

### ISSUE THREAD

**Issue by Kinbaum** [emilkowalski/vaul#102]
On the latest version of vaul (0.6.3) the `[vaul-drawer-wrapper]` is not resetting the transform upon close.

https://github.com/emilkowalski/vaul/assets/6590356/9411df14-e631-4cec-9365-4909069776c8

**Comment by emilkowalski** [comment]
Can you provide a codesandbox? It's working fine on [vaul.emilkowal.ski](https://vaul.emilkowal.ski/)

**Comment by Kinbaum** [comment]
Sure thing. Here you go:
https://codesandbox.io/p/sandbox/funny-pine-8vvtr7

You'll notice the scaling doesn't reset, nor does the background color

**Comment by emilkowalski** [comment]
What's diferent than in the example here? https://codesandbox.io/p/sandbox/drawer-with-scale-g24vvh?file=%2Fapp%2Fmy-drawer.tsx%3A1%2C1

This also runs on the latest version and doesn't have this problem. What did you change from the default example? I don't notice anything that might be causing it.

**Comment by Kinbaum** [comment]
I updated my sandbox to match your example verbatim. The only differences are the versions listed in the package.json. Still doesn’t work

**Comment by raimondlume** [comment]
@emilkowalski should this be a separate issue?

**Comment by suyalcinkaya** [comment]
Also the `border-radius` as well. Had to fix it as below;

```jsx
onClose={() => {
  setTimeout(() => {
    const wrapper = document.querySelector('[vaul-drawer-wrapper]')
    if (wrapper) {
      wrapper.style.transform = ''
      wrapper.style.borderRadius = ''
      document.body.style.background = ''
    }
  }, 0)
}}
```

**Comment by emilkowalski** [comment]
Fixed in #110. Thank you for reporting!

**Comment by emilkowalski** [comment]
@raimondlume That issue you reported has been fixed in #110

**Comment by emilkowalski** [comment]
Fixed in #111
